plugins/net_plugin/net_plugin.cpp

```diff
@@ -2849,21 +2903,21 @@ namespace eosio {
          my_impl->update_chain_info();
          reason = no_reason;
       } catch( const unlinkable_block_exception &ex) {
-         peer_elog(c, "bad signed_block : ${m}", ("m",ex.what()));
+         peer_elog(c, "bad signed_block ${n} : ${m}", ("n", blk_num)("m",ex.what()));
```

Comment:
I think we should also add block_id to all these elogs as well.

Reply:
done